docs/content/Getting-Started/Getting-Started-Docker.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Docker
+permalink: /getting-started/docker
+category: Getting Started
+menuOrder: 2
+redirect_from:
+  - /getting-started-docker
+---
+
+This guide will help you get Cube.js running using Docker.
+
+<!-- prettier-ignore-start -->
+[[info |]]
+| Prefer using Docker Compose?
+| [Check out this page instead](/getting-started/docker/compose).
+<!-- prettier-ignore-end -->
+
+> LOOM VIDEO HERE
+
+## 1. Create a new project
```

Comment:
Maybe "Run Cube.js in Docker" or something. I'm not sure that creating a project is most accurate way to describe the "docker run" step. cc @hassankhan @igorlukanin